recipes/glog/all/conanfile.py

```diff
@@ -18,12 +21,14 @@ class GlogConan(ConanFile):
         "fPIC": [True, False],
         "with_gflags": [True, False],
         "with_threads": [True, False],
+        "with_unwind": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "with_gflags": True,
         "with_threads": True,
+        "with_unwind": True,
```

Comment:
libunwind raises if not Linux or FreeBSD, very bad idea to add this requirement by default regardless of this OS.